src/WinRPM.jl

```diff
@@ -455,15 +469,17 @@ function do_install(package::Package)
     local err = nothing
     for cmd = [`7z x -y $path2 -o$cache`, `7z x -y $cpio -o$installdir`]
         (out, pc) = open(cmd,"r")
-        stdoutstr = readall(out)
+        stdoutstr = readstring(out)
         if !success(pc)
             wait_close(out)
             println(stdoutstr)
             err = pc
-            @unix_only cd(installdir) do
-                if success(`rpm2cpio $path2` | `cpio -imud`)
-                    err = nothing
-                end
+            if is_unix()
+                cd(installdir) do
+                  if success(`rpm2cpio $path2` | `cpio -imud`)
```

Comment:
if your editor is Juno, the mismatch of default indent there relative to the standard julia style is a continued bug that users of that editor should continue reporting to its author